ui/client/components/sidePanels/ScrollbarsExtended.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {PropsWithChildren, useRef, useState, useEffect} from "react"
+import {Scrollbars} from "react-custom-scrollbars"
+import {useDebouncedCallback} from "use-debounce"
+import cn from "classnames"
+import styles from "./ScrollbarsExtended.styl"
+
+export function ScrollbarsExtended({children, onScrollToggle}: PropsWithChildren<{ onScrollToggle?: (isEnabled: boolean) => void }>) {
+  const scrollbars = useRef<Scrollbars>()
+  const [isScrollPossible, setScrollPossible] = useState<boolean>()
+
+  const [onUpdate] = useDebouncedCallback(() => {
+    const {scrollHeight = 0, clientHeight = 0} = scrollbars.current?.getValues() || {}
+    setScrollPossible(scrollHeight - clientHeight > 0)
+  }, 50)
```

Comment:
It would be nice if we would extract 0, 50 to some constants, which names will be more meaningful